### DISCUSSION TITLE
Discussions: Ideas for promoting mobile apps

### DISCUSSION THREAD

**Discussion by nalininsbs** [WordPress/Marketing-Team#476] [Ideas]
Card to pull together discussions on Slack and reference core tickets relating to ideas for promoting mobile apps. 

Thank you to @abhansnuk for assistance with setting this up on GitHub and to @OlaIola for highlighting ideas.

Link to the discussion on the [Make WordPress Marketing Slack](https://wordpress.slack.com/archives/C0GKJ7TFA/p1658576911978589) on 26 July 2022

**Comment by oglekler** [comment]
Many people are using WordPress for personal blogs and companies employees are more mobile now than ever and solve working tasks on a go from cafés and in public transport. To have an ability to comfortably publishing posts and edit content from mobile phone for some of them can be an advantage.

We have good designed Mobile Application but not using it for its full extent.

After attending '[WordPress Portsmouth Online Meetup – 20th July 2022 – WordPress Mobile Apps](https://www.wp-pompey.org.uk/meetup/wordpress-portsmouth-online-meetup-20th-july-2022-wordpress-mobile-apps/)', it becomes obvious that we need to put more efforts in promoting it as our marketing advantage. And the easiest way to start is to add a nice and attractive widget to the Dashboard presenting the Mobile Application.

The dashboard widget will be a reminder for developers that this feature can be offered to users as one more plus to choose WordPress in the first place.

This is an explanation for ticket: [Widget to Dashboard to offer Mobile App to users](https://core.trac.wordpress.org/ticket/56276)

Widget to Dashboard can be a part of new Admin Experience as well: https://make.wordpress.org/design/2022/06/13/thinking-through-the-wordpress-admin-experience/

Right now, Mobile App need XML-RPC to be whitelisted on hosting, otherwise it will not work. There to possibilities: 
1. To make a campaign, starting with hosting companies which are supporting WordPress to whitelist it or
2. Switch application to another protocol (if possible) and avoid this issue all together.
Here is the ticket: [Check hosting compatibility for Mobile Application to work with the site](https://core.trac.wordpress.org/ticket/56276) 

From Marketing point of view Mobile App can be a part of creating new seamless experience for audience when they can be working and communicating through their sites in the same way as with social media - from different devises, from any place and on any time.  And of course if WordPress Admin itself will look like an App, it will help to bring to users up-to-date feelings. At the end of the day it's our feelings what drives our chooses and a good look and helpful UI can create wonders.

"**Go mobile**" can be a campaign with:
1. Social promotions and short tutorials
2. Check list for hostings and hosting users
3. Online events with bloggers who already using Mobile App to work with their sites
4. Feedback for new ideas and suggestions for improvements from all users - simple and nice without need to fill a lot of fields or check if there is a ticket already
 
Let's think big and ambitious, but plain and clear — genius is simple. Write your next post from WordPress Mobile App today! )

**Comment by oglekler** [comment]
Dashboad related tickets:
1. [Improve the Dashboard Screen and Welcome dashboard widget](https://core.trac.wordpress.org/ticket/39617)
2. [Upload custom avatar for user in Dashboard](https://core.trac.wordpress.org/ticket/16020) - 12 years old
3. [New UI for the settings page of the dashboard](https://core.trac.wordpress.org/ticket/38453)
4. [Allow customization of Admin Color Schemes for dashboard](https://core.trac.wordpress.org/ticket/48587)
5. [Add WordPress.tv latest videos dashboard widget](https://core.trac.wordpress.org/ticket/50139)
6. [About page widget to Dashboard](https://core.trac.wordpress.org/ticket/52762) (this is another one of mine)
The obstacle in making people well informed is that most users are not paying attention to the About page, especially if an update was automatically made or someone else is making it for them. Or even if they saw this page, imagine: If I have just updated my site to the latest version, my first priority - to go and check if everything is working and all this useful information including the message about contribution will never catch my attention again, if everything is fine I will go to do what I was about to do before this urge to update.
The desired solution - to make the Dashboard page each time a bit different and interesting, alternative and more simple solution — a widget to Dashboard with a very small piece about a new version with a call to action to read more.

From my point of view Dashboard need a complex approach, and it can be a contact point to connect all developers, users, bloggers and other owners. It's our biggest and easiest point to attract attention and raise awareness. But right now it's boring and people are switching it in an instance to something else.

**Comment by sereedmedia** [comment]
What is the current status on this project/ticket? I am very interested in the development of the mobile app from a WordPress tool perspective, and I think the promotion of it, and hopefully getting user and community feedback about it, is super crucial!

**Comment by abhansnuk** [comment]
Thanks for the question. This is a discussion @sereedmedia started by @OlaIola. There is no timescale or task related to it as yet. You can add comments to the discussion on this discussion ticket. I am sure Olga would welcome your comments and interest too. 

Update:
I have added a discussion label to this too.

**Comment by nalininsbs** [comment]
Highlighted this early stage discussion at the Marketing Team meeting, November 2, 2022.

**Comment by nalininsbs** [comment]
Highlighted at the team meeting on November 9. 2022. @eidolonnight will review card to make some suggestions for next steps before this is shared with the mobile team. Thank you Nicholas. [Link to Slack](https://wordpress.slack.com/archives/C0GKJ7TFA/p1668007032167729).

Added @sereedmedia to the card as has shown interest in working on this.

**Comment by sereedmedia** [comment]
There are two separate posts here submitted by @OlaIola. The first is about developing a campaign to promote the WordPress mobile app and the second is to develop the UI of the WordPress dashboard. 

I think both merit attention and action. I'd like to split them into two separate issues if there are no objectives from @OlaIola or others.

**Comment by oglekler** [comment]
Hi @sereedmedia,
Thank you for pushing this discussion forward.

I created a new ticket #201 for the Dashboard widget, but there are more parts and mentioned things included in the discussion:
1. Promotional campaign itself to raise awareness
2. Input from the Mobile team and possibly the Hosting team about [XML-RPC usage](https://core.trac.wordpress.org/ticket/56276)
3. The idea to change the [Admin experience](https://make.wordpress.org/design/2022/06/13/thinking-through-the-wordpress-admin-experience/)
4. #201
5. Plus an idea to [make About page the first page which person will be seen after an update even if the update was handled by automation such as WordPress itself or user's Hosting provider](https://core.trac.wordpress.org/ticket/57877)

I've posted a call in the Mobile team Slack channel for their update about XML-RPC usage, and I think we need to check the status of tasks for the WordPress Admin Experience, as well as promoting the idea for widget creation in the Core channel (and maybe some others) to get a support and an attention from developers and UI designers.
